### PR TITLE
Added locale detection add-on

### PR DESCRIPTION
**Pull request by phifty**

### CODE REVIEW

**Comment by mholt** [comment]
Cool, thanks for switching it around! I'm going to convert your markdown docs for the website and then I can merge this in and deploy the add-on. :smile:

File: features/registry.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ var Registry = Middlewares{
 	{"hugo", "github.com/hacdias/caddy-hugo", "Powerful and easy static site generator with admin interface."},
 	{"mailout", "github.com/SchumacherFM/mailout", "SMTP client with REST API. Send emails with optional PGP encryption"},
 	{"prometheus", "github.com/miekg/caddy-prometheus", "Prometheus metrics"},
+	{"locale", "github.com/simia-tech/caddy-locale", "Locale detection"},
```
Comment (mholt) [inline features/registry.go]: In [your previous PR](https://github.com/mholt/caddy/blob/c15198c95d9596a8a5314722d581006ce2cafb79/caddy/directives.go#L56) you had this directive after gzip; not at the end. Was this intentional?

Comment (phifty) [inline features/registry.go]: No, it wasn't intentional. Should have read the comment ;-)

**Comment by mholt** [comment]
@phifty Sorry it has been taking me so long to get to this. We're in the process of switching everything over to the Caddy 0.9 plugin format, I'll be submitting a PR or issue on your repo soon!

**Comment by phifty** [comment]
Don't worry. Is there a doc of the new plugin format? If I find time, I can adjust the code.

**Comment by mholt** [comment]
Yeah, on the Caddy wiki: https://github.com/mholt/caddy/wiki/Converting-Add-ons-to-Plugins - it's mostly correct but I might change one thing to fix a slight design flaw before the next 0.9 beta. Still, I don't expect it would be more than ~1 line change, so feel free to go ahead if you find time!

**Comment by phifty** [comment]
Ok thanks, I've changed to code and it should work with 0.9.

**Comment by mholt** [comment]
Cool. I'm still fixing one little design flaw in 0.9 that will affect plugins (just one line I think) so I'll get back to this ASAP!

**Pull request merged**